sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/GroupByKeyTest.java

```diff
@@ -297,7 +288,7 @@ public void testRemerge() {
 
   @Test
   public void testGroupByKeyDirectUnbounded() {
-    Pipeline p = createTestDirectRunner();
+    Pipeline p = TestPipeline.create();
```

Comment:
Don't do it now, because we want to get this in, but this test looks obsolete and/or should now be `RunnableOnService`.